tika-parsers/tika-parsers-standard/tika-parsers-standard-modules/tika-parser-cad-module/src/main/java/org/apache/tika/parser/dwg/DWGParser.java

```diff
@@ -94,6 +93,14 @@ public Set<MediaType> getSupportedTypes(ParseContext context) {
 
     public void parse(InputStream stream, ContentHandler handler, Metadata metadata,
                       ParseContext context) throws IOException, TikaException, SAXException {
+    	
+    	configure(context);
+    	DWGParserConfig dwgc = context.get(DWGParserConfig.class);
+
+if(!dwgc.getDwgReadExecutable().isEmpty()) {
```

Comment:
Do I remember correctly that the dwg commandline handles a different version of files than our existing parser.  If so, we'll want to branch based on detected mime type, not whether the executable is available.

If this is as intended, please ignore this comment.

Reply:
I'm pretty sure DWGRead takes all the versions the current reader does, it only accepts the same mime type as the original DWGParser which is why I've left it here.

Reply:
K.  So the logic is if dwgread is available, we'll skip our current code?  Sounds good!